apps/api/v2/src/modules/users/validators/avatarValidator.ts

```diff
@@ -1,11 +1,32 @@
 import { ValidatorConstraint } from "class-validator";
 import type { ValidatorConstraintInterface } from "class-validator";
 
+const BASE64_IMAGE_REGEX =
+  /^data:image\/[^;]+;base64,(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$/;
+
 @ValidatorConstraint({ name: "avatarValidator", async: false })
 export class AvatarValidator implements ValidatorConstraintInterface {
-  validate(avatarString: string) {
-    // Checks if avatar string is a valid base 64 image
-    const regex = /^data:image\/[^;]+;base64,(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$/;
-    return regex.test(avatarString);
+  validate(avatarString: string): boolean {
+    if (avatarString === null || avatarString === undefined) return true;
```

Comment:
nit: TS confirms at line 9 that avatarString can be string only.  
So, we just need  
if (!avatarString?.trim()) return false;